documentation/configuration.md

```diff
@@ -237,18 +236,6 @@ Additional metrics providers and counter names to return from this route can be
 }
 ```
 
```

Comment:
```suggestion

> **Warning:** In the default configuration, custom metrics will be exposed along with all other metrics on an unauthenticated endpoint. If your metrics contains sensitive information, we recommend disabling the [metrics urls](#metrics-urls) and consuming metrics from the authenticated endpoint (`--urls`) instead.

```

Reply:
I don't see a "metrics-urls" section. Did you intend to link to something else?

Reply:
Nevermind, found it!

Reply:
I would not recommend disabling metrics url, but manually upgrading it. Remember the metrics url has a more permissive binding but a more restricted set of actions.

Reply:
Do we automatically auth upgraded endpoints?

Reply:
If we do, I think that's a bad idea. If we don't, it doesn't help prevent disclosing sensitive information.
Either way, I got issues with it 😂

Reply:
We do not but I see what you are getting at.

Reply:
I logged another issue where we can allow adding auth to the metrics URLs (Prometheus does seem to support specifying the Authorization header) but we would likely want to be able to configuration the authentication separately from the default one. See https://github.com/dotnet/dotnet-monitor/issues/216